src/app/screens/Onboard/TestConnection/index.tsx

```diff
@@ -49,11 +49,13 @@ export default function TestConnection() {
       setLoading(false);
       clearTimeout(timer);
     }
-  }
+  }, [t]);
```

Comment:
why was this not needed before? 
did you test this very well? (I had strange errors in some places when I had to do this callback,useEffect stuff - and I still don't understand it) (e.g. did you test the timeout?)
you are missing the test connection for the options page.

Reply:
> why was this not needed before?

Define "before"

> did you test this very well? (I had strange errors in some places when I had to do this callback,useEffect stuff - and I still don't understand it) (e.g. did you test the timeout?) 

Let me test. Just opened this to see if the pipeline will be green.

> you are missing the test connection for the options page.

What do you mean? With this PR I just wanted to fix the warning.

Reply:
please always make sure that things work when you change thing on such critical components. They simply have to work, if it's broken for some reason then people will have issues adding accounts. Checking the pipeline is (and will never be) enough.
This also helps you to understand what goes on and how things are used. 

So here for example you seem to miss that there are two similar flows (welcome + options) both have a test-connection screen - and sadly are not very DRY. So you have to make this change in the other screen, too.

Reply:
Tested the welcome/onboard testconnection with this and it seems to work well.  
The same way is not working for the options screen so I'm leaving that for now and will have a look with Lisa when she's back.
Would you mind testing this as well?